server/controllers/clap.controller.js

```diff
@@ -0,0 +1,38 @@
+import db from '../models';
+import Response from '../utils/response.util';
+
+const { Clap, Article } = db;
+/**
+ * Clap Controller
+ * @package clap
+ */
+class ClapController {
+  /**
+   * @description - Controller for create a clap endpoint
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} response
+   * @memberof createClap
+  */
+  static async createClap(req, res) {
+    const {
+      articleId
+    } = req.body;
+    const { id } = req.user;
+    const successMessage = 'Clap successfully created';
+    const errorMessage = 'Clap was not created successfully';
+    const userCantClapTwice = await Clap.findOne({ where: { articleId, userId: id } });
+    if (userCantClapTwice != null) return Response(res, 400, 'you cannot clap twice');
+    const cantClapSelf = await Article.findOne({ where: { userId: id }, as: 'user' });
```

Comment:
The variable name should be improved upon. You can use something like `userOwnArticle`